projects/contracts/src/lib/media.types.ts

```diff
@@ -1,6 +1,7 @@
-import {Dictionary} from "./types";
+import { Dictionary } from "./types";
 
 export enum MediaType {
+  Invalid = -1,
```

Comment:
oh was it needed? lets hope nothing breaks haha

Reply:
I mean I just wanted to have a common id for the invalid group and not some random 1337 number, that someone will eventualle use in that enum, cause thats when things start breaking.

Reply:
maybe I should add more "defaults" to the enums so in that case its good